### PR TITLE
[ISSUE #427]feat(replicator):add consumer offset sync

### LINKED ISSUE #427: consumer offset sync to target cluster
**FEATURE REQUEST**

1. Please describe the feature you are requesting.
consumer offset on src cluster should sync to target cluster.

2. Provide any additional detail on your proposed use case for this feature.

2. Indicate the importance of this issue to you (blocker, must-have, should-have, nice-to-have). Are you currently using any workarounds to address this issue?

4. If there are some sub-tasks using -[] for each subtask and create a corresponding issue to map to the sub task:

- [sub-task1-issue-number](example_sub_issue1_link_here): sub-task1 description here, 
- [sub-task2-issue-number](example_sub_issue2_link_here): sub-task2 description here,
- ...

### PR DESCRIPTION
**Pull request by xujianhai666**
sync src cluster consumer groups offset to message store

- Add MetaSourceConnector as sourceConnector for task assign
- Add MetaSourceTask as SourceTask for offset read and message sink
- Add RmqConnectorConfig for config info 

Closes #427

### CODE REVIEW

File: rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceConnector.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
+package org.apache.rocketmq.replicator;/*
```
Comment (duhenglucky) [inline rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceConnector.java]: the package should close before import

File: rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceConnector.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+import org.apache.rocketmq.replicator.common.Utils;
+import org.apache.rocketmq.replicator.config.ConfigDefine;
+import org.apache.rocketmq.replicator.config.DataType;
+import org.apache.rocketmq.replicator.config.TaskDivideConfig;
+import org.apache.rocketmq.replicator.strategy.TaskDivideStrategy;
+import org.apache.rocketmq.tools.admin.DefaultMQAdminExt;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MetaSourceConnector extends SourceConnector {
```
Comment (duhenglucky) [inline rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceConnector.java]: [Discuss] Should the metadata connector be integrated with the message connector for easy management?

File: rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceConnector.java (hunk 1 above)
Comment (xujianhai666) [reply rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceConnector.java]: done @duhenglucky

File: rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceConnector.java (hunk 2 above)
Comment (xujianhai666) [reply rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceConnector.java]: @duhenglucky done

File: rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceTask.java [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    @Override public Collection<SourceDataEntry> poll() {
+        log.debug("polling...");
+        List<String> groups = JSONObject.parseArray(this.config.getTaskGroupList(), String.class);
+
+        if (groups == null) {
+            log.info("no group in task.");
+            try {
+                Thread.sleep(TimeUnit.SECONDS.toMillis(10));
+            } catch (InterruptedException e) {
+                e.printStackTrace();
```
[the remark this answers is not in the document]
Comment (xujianhai666) [reply rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/MetaSourceTask.java]: @odbozhou done

**Pull request merged**